### DISCUSSION TITLE
sprites file

### DISCUSSION THREAD

**Discussion by Iheff** [dicebear/dicebear#128] [Q&A] [answered]
Hi Dicebear team, I was wondering how the sprite.gvdesign file in the human male/female avatars projects is used (if other than a reference) to create the paths used in the svg generation?

I am keen to create a set of avatars using the sprite template but when extracting these paths in gravit designer I couldnt match the paths in the files (which appear to be compressed versions), so wondered if there was a scripted process (couldnt find) or if this was generally a manual step.
Thanks.

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @Iheff ,

the paths for the styles "male" and "female" were once created and exported with [Gravit Designer](https://designer.gravit.io/). The two files are now only used for reference and are not actively used. However, you can still use them to recreate the generated avatars.

<img width="575" alt="Bildschirmfoto 2021-06-19 um 17 20 10" src="https://user-images.githubusercontent.com/647303/122647049-9bec9b00-d122-11eb-9192-37d2581eae5c.png">

A comparison with the paths in the JavaScript files is probably no longer possible because we always try to keep the files as small as possible and have reduced the export from Gravit first by [SVGO](https://jakearchibald.github.io/svgomg/).

There is no automated process. We export the paths, compress them with SVGO and then copy them manually into the JavaScript file.

For newer avatar styles we use [Figma](https://www.figma.com/) in the meantime. But the process is still manual there, although the compression is already done by Figma.

**Comment by Iheff** [comment]
Thanks so much for your response, and added details. I will take a look at figma as a design tool for pre-compressed path generation.

It would be very interesting if I could involve a Base graphic File to somehow export a style from, if I have any luck in that area I will loop back round and add a function/command  to your CLI tool but for now I’ll try to just export some new paths via figma.

Thanks again much appreciated
